packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.tsx

```diff
@@ -54,32 +56,48 @@ export class AtomicResultPrintableUri {
     return (
       <li part="result-printable-uri-list-element">
         <button
-          part="result-printable-uri-list-ellipsis"
+          part="result-printable-uri-list-ellipsis align-middle"
           aria-label={this.strings.collapsedUriParts()}
           onClick={() => (this.listExpanded = true)}
         >
           ...
         </button>
+        {this.renderSeparator()}
       </li>
     );
   }
 
   private get allParents() {
     const parentsXml = parseXML(`${this.result.raw.parents}`);
     const parents = Array.from(parentsXml.getElementsByTagName('parent'));
-    return parents.map((parent) => {
+    return parents.map((parent, i) => {
       const name = parent.getAttribute('name');
       const uri = parent.getAttribute('uri')!;
       return (
         <li part="result-printable-uri-list-element">
-          <a part="result-printable-uri-link" href={filterProtocol(uri)}>
+          <a
+            part="result-printable-uri-link"
+            class="text-sm inline-block align-middle"
+            href={filterProtocol(uri)}
+          >
             {name}
           </a>
+          {i === parents.length - 1 ? null : this.renderSeparator()}
         </li>
       );
     });
   }
 
+  private renderSeparator() {
+    return (
+      <span
```

Comment:
In your screenshot, the contrast between separators and the background looks a little low. What's the contrast ratio for it?